mvlearn/utils/utils.py

```diff
@@ -155,3 +172,345 @@ def check_Xs_y_nan_allowed(
              found {}".format(num_classes, n_classes))
 
     return Xs_converted, y_converted
+
+
+class linear_cca():
```

Comment:
How is this different from kcca with a linear kernel or gcca with 2 views? I would avoid replicating existing code (even tho at this point it'll take longer for you to change your code to use an existing implementation than keep as is).

Reply:
fyi going to skip code review for this class until this comment is resolved.

Reply:
The new kcca hasn't been finished yet, so I don't have the code for that. This is the implementation of CCA that I got from the github that this DCCA is based off of. Once we have the updated KCCA pushed in, then I can switch to that, but until then I thought we could just use this basic CCA with no kernel.